### ISSUE TITLE
Divider custom example does not load in AMD

### ISSUE THREAD

**Issue by joechung-msft** [OfficeDev/office-ui-fabric-react#5711]
### Bug Report 
 
- __Package version(s)__: 6.41.0
- __Browser and OS versions__: 
 
#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No) No
 
Requested priority: (Blocking, High, Normal, Low) Normal
 
Products/sites affected: (if applicable)
 
#### Describe the issue:
 
The Divider custom example does not load when loaded as an AMD module with require.js.

The Divider import in VerticalDivider.Custom.Example should be:
`import { VerticalDivider, IVerticalDividerClassNames } from 'office-ui-fabric-react/lib/Divider';`
 
#### Actual behavior:
 
Uncaught JavaScript error thrown by require.js:

- `Uncaught Error: Script error for "office-ui-fabric-react/lib/components/Divider", needed by: office-ui-fabric-react/lib/components/Divider/examples/VerticalDivider.Custom.Example`
 
#### Expected behavior:
 
The example should load and render correctly.
 
### If applicable, please provide a repro:

````html 
<!DOCTYPE html>
<html>
<head>
    <meta charset="utf-8">
    <title></title>
    <link rel="stylesheet" href="https://unpkg.com/office-ui-fabric-react@6.41.0/dist/css/fabric.min.css">
    <script src="https://unpkg.com/babel-polyfill/dist/polyfill.min.js"></script>
    <script src="https://unpkg.com/babel-standalone/babel.min.js"></script>
</head>
<body>
    <div id="app"></div>
    <script src="https://unpkg.com/requirejs/require.js"></script>
    <script type="text/babel" data-presets="es2015,react">
        require.config({
            packages: [
                { name: '@microsoft/load-themed-styles', location: 'https://unpkg.com/@microsoft/load-themed-styles/lib-amd', main: 'index.js' },
                { name: '@uifabric/icons', location: 'https://unpkg.com/@uifabric/icons/lib-amd', main: 'index.js' },
                { name: '@uifabric/merge-styles', location: 'https://unpkg.com/@uifabric/merge-styles/lib-amd', main: 'index.js' },
                { name: '@uifabric/styling', location: 'https://unpkg.com/@uifabric/styling/lib-amd', main: 'index.js' },
                { name: '@uifabric/utilities', location: 'https://unpkg.com/@uifabric/utilities/lib-amd', main: 'index.js' }
            ],
            paths: {
                'office-ui-fabric-react/lib': 'https://unpkg.com/office-ui-fabric-react@6.41.0/lib-amd',
                'prop-types': 'https://unpkg.com/prop-types/prop-types.min',
                'react': 'https://unpkg.com/react/umd/react.production.min',
                'react-dom': 'https://unpkg.com/react-dom/umd/react-dom.production.min',
                'tslib': 'https://unpkg.com/tslib/tslib'
            }
        })([
            'react',
            'react-dom',
            'office-ui-fabric-react/lib/components/Divider/examples/VerticalDivider.Custom.Example',
            'office-ui-fabric-react/lib/Fabric',
            'office-ui-fabric-react/lib/Icons'
        ], (
            React,
            { render },
            { VerticalDividerCustomExample: DividerCustomExample },
            { Fabric },
            { initializeIcons }
        ) => {
            initializeIcons();

            function App() {
                return (
                    <Fabric>
                        <DividerCustomExample />
                    </Fabric>
                );
            }

            render(<App />, document.getElementById('app'));
        });
    </script>
</body>
</html>
````

**Comment by joechung-msft** [comment]
There is a similar issue with the DetailsList import in DetailsList.Shimmer.Example.  It should be:
`import { DetailsListLayoutMode, IColumn } from 'office-ui-fabric-react/lib/DetailsList';`

**Comment by cliffkoh** [comment]
@JasonGore I could take a closer look but is this and this class of issues fixed with your checkin?

**Comment by cliffkoh** [comment]
Hmm it is still broken in master :\

**Comment by cliffkoh** [comment]
Co-assigning to @JasonGore because he has done a deep investigation of this today. Jason, please feel free to let us know if you will not have cycles to look into it.

**Comment by JasonGore** [comment]
No, I only addressed import matching issues. It look like the script is currently written to only check relative paths, not absolute paths like package imports, so it doesn't find this class of issue.

**Comment by cliffkoh** [comment]
@joechung-msft fixed in #5790. Thanks for reporting.
